gluten-substrait/src/main/scala/org/apache/gluten/utils/SubstraitUtil.scala

```diff
@@ -107,4 +109,20 @@ object SubstraitUtil {
     val nameList = ConverterUtils.collectAttributeNamesWithExprId(output)
     createNameStructBuilder(typeList, nameList, Collections.emptyList()).build()
   }
+
+  def convertJavaObjectToAny(obj: AnyRef): Any = {
+    if (obj == null) return null
+    var msg: Message = null
```

Comment:
suggestion: 
```
val msg = obj match {
    ***
}
```

Reply:
I had to leave in the `: Message` otherwise I got a compiler error I think because the types in the different cases are different, but otherwise made the change.